ocaml/xapi/xapi_vdi.mli

```diff
@@ -0,0 +1,200 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(** Checks to see if an operation is valid in this state. Returns Some exception
+    if not and None if everything is ok. *)
+(* Exposed here only for use by Test_vdi_allowed_operations, this
+   declaration also serves to annotate the (op:API.vdi_operations)
```

Comment:
It's fine as it is, but I wouldn't mention the second part, because that's too much details in my opinion.